Northstar.Client/mod/scripts/vscripts/ui/menu_ns_serverbrowser.nut

```diff
@@ -967,49 +967,80 @@ void function OnServerSelected_Threaded( var button )
 	bool autoDownloadAllowed = GetConVarBool( "allow_mod_auto_download" )
 	int downloadedMods = 0;
 
+	// Check out if there's any server-required mod that is not locally installed
+	array<string> modNames = NSGetModNames()
+	bool uninstalledModFound = false
+	foreach ( requiredModInfo in server.requiredMods )
+	{
+		if ( !modNames.contains( requiredModInfo.name ) )
+		{
+			uninstalledModFound = true
+			break
+		}
+	}
```

Comment:
Maybe I'm missing something here but wouldn't that skip the mod download if an older version of the mod is installed?

Reply:
That is right (and unexpected); I tried joining a `Parkour` server needing `v0.2.1` while I had `v0.1.2`: auto-download did not trigger, my client tried to join the server and got a script error (due to the script differences between `v0.2.1` and `v0.1.2`).
Gotta fix that!

Reply:
So this has been fixed, meaning `v0.2.1` is downloaded even if you locally have `v0.1.2`.
I however ran into another issue, which is Northstar doesn't know how to handle multiple versions of the same mod.

Reply:
> I however ran into another issue, which is Northstar doesn't know how to handle multiple versions of the same mod; this is unrelated to this feature, but the PR depends on it...

Are you planning to fix the underlying issue first? Cause I would've been fine with merging the PR despite this issue being unfixed as it persisted before already anyway and this PR just uncovers it. So as long as we fix it before re-enabling MAD I'm fine with either order of merging PRs ^^

Reply:
If you're fine with merging this first, I'm fine with it too!